EditPortfolio.js

```diff
@@ -46,3 +53,32 @@ function clearFormInputs() {
   inputEndDate.value = "";
   inputDescription.value = "";
 }
+
+function updateLocalStorage(newExperience) {
+  const experiences = JSON.parse(localStorage.getItem("experiences")) || [];
+
+  experiences.push(newExperience);
+
+  localStorage.setItem("experiences", JSON.stringify(experiences));
+}
+
+document.addEventListener("DOMContentLoaded", function () {
+  loadExperiencesFromLocalStorage();
+});
+
+function loadExperiencesFromLocalStorage() {
```

Comment:
It would be better if we move the functions into separate modules to reduce repetition and it would be easy to reuse them.